source/getting_started.rst

```diff
@@ -96,14 +135,28 @@ This is done by typing ``path`` in the MATLAB Command Window::
 
 
 The WEC-Sim source directory, ``$WECSIM/source``, and its subfolders should appear in this list. 
-	
-	
+
+.. note::
+    This works only after a restart. Will I always have the wecSimPath variable 
+    included in my default workspace?
+
 Step 2. Add WEC-Sim Library to Simulink
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 Open the Simulink Library Browser by typing ``slLibraryBrowser`` in the MATLAB Command Window::
 
 	>> slLibraryBrowser
 
+.. note::
+    This generates this warning a number of times for me::
+    
+        Warning: Cannot find library called 'WECSim_Lib_2015b'.
+        > In LibraryBrowser.internal/RepositoryGenerator/buildTree
+          In LibraryBrowser.internal/RepositoryGenerator/buildTree
+          In LibraryBrowser.internal/RepositoryGenerator/generate
+          In LibraryBrowser.internal.generateRepository 
+    
+    Is that normal?
+
```

Comment:
@H0R5E this was due to a WEC-Sim Library issue we have now resolved. You should no longer get these warnings.